specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3535,6 +3535,125 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/ipAddressAggregates": {
+      "get": {
+        "tags": [
+          "Reports"
+        ],
+        "description": "Gets the ipAddressAggregates for a given service.",
+        "x-ms-examples": {
+          "services_listIpAddressAggregates": {
+            "$ref": "./examples/IpAddressAggregates.json"
+          }
+        },
+        "operationId": "services_listIpAddressAggregates",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "skiptoken",
+            "in": "query",
+            "description": "Used to paginate results",
```

Comment:
nit: Would be good to indicate where the value is from. For example could be: "A returned in paginated result continuation token to load different pages. "

Reply:
Fixed